lambdas/initialise-ipv-session/src/test/java/uk/gov/di/ipv/core/initialiseipvsession/InitialiseIpvSessionHandlerTest.java

```diff
@@ -591,12 +591,14 @@ void shouldSendAuditEventForIpvInheritedIdentityVcReceived() throws Exception {
             var restricted =
                     (AuditRestrictedInheritedIdentity) inheritedIdentityAuditEvent.getRestricted();
             assertEquals(
-                    "[{\"nameParts\":[{\"type\":\"GivenName\",\"value\":\"KENNETH\"},{\"type\":\"FamilyName\",\"value\":\"DECERQUEIRA\"}]}]",
-                    restricted.name().toString());
-            assertEquals("[{\"value\":\"1965-07-08\"}]", restricted.birthDate().toString());
+                    "[{\"nameParts\":[{\"value\":\"KENNETH\",\"type\":\"GivenName\"},{\"value\":\"DECERQUEIRA\",\"type\":\"FamilyName\"}]}]",
```

Comment:
As we're now using Jackson, the ordering we set for fields in the `NameParts` class is now being respected, so this is just an ordering change.

Reply:
I did wonder about this as I seem to remember it was me that set that ordering ages ago, for some reason. The examples in the vocab would seem to suggest value, then type: https://vocab.account.gov.uk/v1/examples/NameClass/OneGivenOneFamily.json
I'm not fussed really, happy to switch it back again if you think

Reply:
:+1: matching the Vocab seems like as good a directive as any!